app.authpass.AuthPass.yaml

```
@@ -0,0 +1,61 @@
+app-id: app.authpass.AuthPass
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: authpass
+finish-args:
+  # Filesystem and network access
+  - --filesystem=home
+  - --share=network
+  # Display access
+  - --socket=wayland
+  - --share=ipc
+  - --socket=fallback-x11
+  # Keyring access
+  - --talk-name=org.freedesktop.secrets
+  # OpenGL rendering
+  - --device=dri
+
+cleanup:
+  - /include
+
+modules:
+  - name: libkeybinder
+    buildsystem: autotools
+    sources:
+      - type: archive
+        url: https://github.com/kupferlauncher/keybinder/releases/download/keybinder-3.0-v0.3.2/keybinder-3.0-0.3.2.tar.gz
+        sha256: e6e3de4e1f3b201814a956ab8f16dfc8a262db1937ff1eee4d855365398c6020
+
+  - name: authpass
+    buildsystem: simple
+    build-commands:
+      - tar -zxf authpass-linux-latest.tar.gz
+      - cp -r authpass/* /app/
+      - chmod +x /app/authpass
+      - mkdir -p /app/bin
+      - ln -s /app/authpass /app/bin/authpass
+      - install -Dm644 authpass-git/metadata/linux/app.authpass.AuthPass.png /app/share/icons/hicolor/512x512/apps/app.authpass.AuthPass.png
+      - install -Dm644 authpass-git/metadata/linux/app.authpass.AuthPass.desktop /app/share/applications/app.authpass.AuthPass.desktop
+      - install -Dpm644 authpass-git/metadata/linux/app.authpass.AuthPass.metainfo.xml
+        /app/share/metainfo/app.authpass.AuthPass.metainfo.xml
+    sources:
+      - type: file
```

if you made this 
```suggestion
      - type: archive
        dest: authpass
```

You wouldn't need to extract the archive.